Documentation/PackedAssetBuilds.md

```diff
@@ -18,47 +46,94 @@ Inside the installer select the version to use.  Install version **2021.1.11.793
 
 > REBOOT (or logout / login). Otherwise, the environment settings won't get picked up for any builds in Visual Studio. They will only apply to command-line builds, and only for any command-line windows that have been opened after the installer finishes.
 
-### Build profile build and process assets
+### Build non-monolithic profile builds of the Engine
 
-Build and run MPS as per the [README.md](../README.md) and ensure all assets are built.
+Build a regular profile build of the game as per the [README.md](../README.md) in an engine-centric way. Be sure to build from the source engine and not the installer. Here is an example:
 
-### Test the profile build
+Clone the engine from source.
 
-* Open the game in editor
-    * load `NewStarBase` level
-    * Verify that game can launch and connect to local server from editor
-* Validate local game launcher can connect to local server
+```shell
+C:\git> git clone https://github.com/o3de/o3de
+```
+
+Navigate to `C:\git\o3de`.
+Create a build folder for non-monolithic build of the engine, for example `C:\git\o3de\build_non_mono`.
+
+```shell
+C:\git\o3de> mkdir build_non_mono
+```
+> The location and the name of the build folder is optional.
+
+Configure the engine in a non-monolithic mode with o3de-multiplayersample project.
+
+```shell
+C:\git\o3de\build_non_mono> cmake .. -DLY_MONOLITHIC_GAME=OFF -DLY_PROJECTS="C:\git\o3de-multiplayersample"
```

Comment:
Well this is missing the change directory call to build_non_mono.
Though I see the old steps as superior as it uses the -B and -S option to avoid the need of additional change directroy call.

```suggestion
C:\git\o3de> mkdir build_non_mono
```
> The location and the name of the build folder is optional.

Configure the engine in a non-monolithic mode with o3de-multiplayersample project.

```shell
C:\git\o3de\> cmake -B build_non_mono -S. -DLY_MONOLITHIC_GAME=OFF -DLY_PROJECTS="C:\git\o3de-multiplayersample"
```

Reply:
OK